Gemfile

```diff
@@ -12,7 +12,7 @@ gem 'aws-sdk-rails', '~> 3'
 gem 'aws-sdk-s3', '~> 1'
 
 gem 'dotenv-rails', '~> 2.7'
-gem 'rails', '~> 5.2.7'
+gem 'rails', '~> 5.2.8'
```

Comment:
PR look good to me. Was changing this necessary? Not a problem. Just asking.

Reply:
It was not, but I thought it would be less confusing :wink: